### ISSUE TITLE
using no parameters in device action

### ISSUE THREAD

**Issue by pixelflock** [googlesamples/assistant-sdk-python#178]
Hello,

I am attempting to write some custom device actions using the google-assistant-sdk python library. For some custom traits the user provides parameters via query patterns and for others the query pattern contains no parameters. I am having issues when I am not using any parameters. in my action.json file I have the following for a trait that requires no parameters.  

```
"intent": {
          "name": "addLight",
          "trigger": {
            "queryPatterns": [
              "Add light"
              ]
            }
          },
          "directResponseFulfillment": {
            "ttsPattern": "adding light"
          },
          "requiredTraits": ["light"]
        }
```

I am getting an error when I add the trait in the sample hotword.py:

```
if event.type == EventType.ON_DEVICE_ACTION:
        for command, params in process_device_actions(event, device_id):
                if command == "addLight":
                        print('Attempting to add light')
                        send_osc( "/lights/add" );
```

The error I get is : 
line 74, in process_device_actions
    if e['params']:
KeyError: 'params'

Basically it is saying that there is no params. And that is correct because I do not want the user to provide any parameters. Is there anyway to assign null to the parameters from the device action json file. If someone can give an example of the python side of the api regarding how to add traits without any parameters, it would be really helpful. Please let me know if you need further explanation. Thanks and regards.

**Comment by Fleker** [comment]
Right now custom device actions are not generally available. With regards to the bug itself, you should be able to modify `hotword.py` to not include params. We will fix this bug in the future.

```python
   if params in e:
     yield e['command'], e['params']
   else:
     yield e['command'], None
```

**Comment by sandblasted7** [comment]
Thank you! That change in the if statement took care of the error. Regards.
